backend/core/src/migration/1646422264825-addCommunityPrograms.ts

```diff
@@ -0,0 +1,35 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class addCommunityPrograms1646422264825 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const [{ id }] = await queryRunner.query(`SELECT id FROM jurisdictions WHERE name = 'Detroit'`)
+
+        await queryRunner.query(
+            `INSERT INTO programs (title)
+                VALUES
+                    ('Seniors 55+'),
+                    ('Seniors 62+'),
```

Comment:
Spreadsheet also contains just  `Seniors` e.g. row 19 . Is it a separate program or just a mistake?

Reply:
I think it may be a mistake